src/services/repo/hooks.spec.tsx

```diff
@@ -31,15 +32,20 @@ const wrapper =
 const server = setupServer()
 
 beforeAll(() => server.listen())
-afterEach(() => server.resetHandlers())
+afterEach(() => {
+  queryClient.clear()
+  server.resetHandlers()
+})
 afterAll(() => server.close())
 
+console.error = () => {}
```

Comment:
Can you mock this console in a setup, but also reset it after the test is finished, just to ensure that the console is back and running normally.

Reply:
Yep yep! Fixed in [5ef5686](https://github.com/codecov/gazebo/pull/2862/commits/5ef5686ed967ef9475f3562b92ddb03847bef5e3)